mlflow/server/otel_api.py

```diff
@@ -77,8 +77,19 @@ async def export_traces(
     parsed_request = ExportTraceServiceRequest()
 
     try:
+        # In protobuf 5.x, ParseFromString may not raise DecodeError on invalid data
```

Comment:
Are we referring to the version of the Python protobuf library? The maximum proto version is 3, but the protobuf Python library latest versions is 6.32.0, not 5.x. Can we calrify in the comment?

Reply:
Updated for clarity